tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -1,9 +1,11 @@
 -   case: from_queryset_with_base_manager
     main: |
         from myapp.models import MyModel
-        reveal_type(MyModel().objects)  # N: Revealed type is "myapp.models.MyModel_NewManager[myapp.models.MyModel]"
+        reveal_type(MyModel().objects)  # N: Revealed type is "myapp.models.NewManager[myapp.models.MyModel]"
```

Comment:
This is since the code no longer has to generate a new version of the manager with a type var, since it now already has one.